### PR TITLE
fix(providers): added auto close for quotes and brackets for tss

### PR DESCRIPTION
**Pull request by longton95**
Added auto closing for quotes and brackets in the tss file and modified the style auto complete to
behave the same for know tags

### CODE REVIEW

File: src/providers/completion/styleCompletionItemProvider.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class StyleCompletionItemProvider implements CompletionItemProvider {
 		} else if (/^['"][.#]\w*$/.test(linePrefix)) {
 			return this.getClassOrIdCompletions(linePrefix, prefix);
 			// tag - "Wind_ or "_
-		} else if (/^['"]\w*$/.test(linePrefix)) {
+		} else if (/^['"]\w['"]*$/.test(linePrefix)) {
```
Comment (ewanharris) [inline src/providers/completion/styleCompletionItemProvider.ts]: I think this regex change needs to be mirrored to the other checks as I'm not getting suggestions based off classes/ids from the view files

File: src/providers/completion/styleCompletionItemProvider.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ export class StyleCompletionItemProvider implements CompletionItemProvider {
 					label: key,
 					kind: CompletionItemKind.Class,
 					detail: value.apiName,
-					insertText: new SnippetString(`${key}": {\n\t\${1}\t\n}`)
+					insertText: key
```
Comment (ewanharris) [inline src/providers/completion/styleCompletionItemProvider.ts]: Is there a way we can keep this snippetstring behaviour where it takes a user down to the next line?

**Comment by hansemannn** [comment]
Good improvement! Any timeline on shipping this?

File: src/providers/completion/styleCompletionItemProvider.ts [hunk 3]
```diff
@@ -21,37 +21,36 @@ export class StyleCompletionItemProvider implements CompletionItemProvider {
...
-					ruleResult = await rule.getCompletions();
-				}
-			}
-			if (ruleResult) {
-				return ruleResult;
-			} else {
-				return this.getPropertyValueCompletions(linePrefix, prefix);
+		if (/\s*\w+\s*:\s*\w*[(]?["'.]?\w*["'.]?[,]?$/.test(linePrefix)) {
+		// first attempt Alloy rules (i18n, image etc.)
+		let ruleResult;
```
Comment (ewanharris) [inline src/providers/completion/styleCompletionItemProvider.ts]: This indentation looks a little off

**Comment by ewanharris** [comment]
@hansemannn, it's pretty much good to go so I imagine we'll ship this mid-late next week

**Pull request merged**